### ISSUE TITLE
Demo

### ISSUE THREAD

**Issue by HDVinnie** [PHPJunior/laravel-video-chat#9]
Is there a demo of this available?

**Comment by MatthewMelvinBelcher** [comment]
ive not been able to get sockets working or i would have a working demo

**Comment by vesper8** [comment]
There is a demo right here https://github.com/PHPJunior/laravel-video-chat-demo

However I have not been able to make it work. If you want to use the WebRTC functionality it looks like you have to set up your own ICE server but there is no explanation on how to do that.

Also despite putting in my pusher credentials and getting my own laravel-echo-server running, I could not get realtime chat to work.

Basically this codebase looked very promising but it's missing some vital instructions and I was unable to find any tutorial online referencing this repo... really wish the author @PHPJunior took the time to write instructions on how to get it fully working in a reproducible manner
